_component_design/time-picker.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Time Picker
+parent: Inputs and Controls
+layout: component
+category: Components
+usage: Enable users to select a specific time or window from a sequential list. The Time Picker is often accompanied by a date picker for the purpose of creating a date range filter.
+preview-image: preview-images/time-picker.svg
+resource: true
+last-modified: 2017-08-17
+---
+
+{% include toc.html %}
+
+## Introduction
+
+<div class="hxRow">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+### When to use
+Use the time picker if the list of times are sequential, such as 9am, 10am, 11am, and so on.
```

Comment:
...the list of times is sequential.. .
Write times as "9 a.m.", "10 a.m.", ... (space after numeral and {a.m., p.m.})
https://rackerlabs.github.io/docs-rackspace/style-guide/style/time.html

Reply:
(applies throughout doc)